errors.ts

```diff
@@ -77,17 +77,15 @@ export function installUncaughtExceptionListener(): void {
 }
 
 export class Errors implements IErrors {
-	constructor(
-		private $logger: ILogger,
-		private $config: Config.IConfig) {
-	}
+
+	_printCallStack: boolean = false;
+	public get printCallStack(): boolean { return this._printCallStack; }
+	public set printCallStack(newVal: boolean) { this._printCallStack = newVal; }
```

Comment:
Do we need `get` and `set`. Just `public printCallStack = false` should be enough